test/routing/Page.test.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import { BrowserRouter, Switch, Route, Link } from 'react-router-dom';
+import Page from '../../src/routing/Page';
+
+describe('Routing - Page', () => {
+  test('can switch pages', () => {
```

Comment:
```suggestion
  test('registers the page route with the router', () => {
```
Or even
```suggestion
  test('is navigable', () => {
```
Because the page isn't really in charge of "switching pages." What it does it render a `Route` and the magic happens under the hood.